docs/File-Formats.md

```diff
@@ -1310,3 +1311,56 @@ The cells contain treatment response values that can be postive and negative rea
 <tr><td>AEW541</td><td>Larotrectinib</td><td>TrkA/B/C inhibitor</td><td>https://en.wikipedia.org/wiki/Larotrectinib</td><td>>8</td><td>2.33</td><td>2.68</td></tr>
 <tr><td>AZD0530</td><td>Saracatinib</td><td>Src/Bcr-Abl inhibitor</td><td>https://en.wikipedia.org/wiki/Saracatinib</td><td>NA</td><td>>8</td><td>4.60</td></tr>
 </table>
+
+## Generic Assay
+Generic assay is a generic data type, which can holds different types of data on samples. cBioPortal supports any number of response data types imported from separate data files. Any meta information on generic assay are not part of the cBioPortal seed database, but are imported automatically from generic assay data files. For meta information already present in the database, all the fields are overwitten by subsequent imports of new generic assay data files. The `entity_stable_id` field must be unique within a data file.
```

Comment:
Can we change to:

Generic Assay is a two dimensional matrix generalized to capture non-genetic measurements per sample.  Instead of a gene per row and a sample per column, a Generic Assay file contains a generic entity per row and a sample per column.  A generic entity is defined by the data curator and generally means something other than a gene.  Some examples include, treatment response or mutational signatures.  For each generic entity - sample pair, a real number represents a captured measurement.